tests/test_decoration_contents.py

```diff
@@ -1,37 +1,3 @@
-def test_tuple_of_validator():
-    from qmxgraph import decoration_contents
-    import attr
-    import pytest
-
-    @attr.s
-    class Cutlery:
```

Comment:
Strangely these local classes were crashing mypy... I was investigating it when I noticed that this is actually testing `attrs` itself, probably written as an exploration aid to learn about attrs validations back in 2017), and left here.

Just removed it then.

Reply:
Actually this was testing if our custom validator interacts OK with attrs.

Reply:
Ahh good to know, will bring the test back then, thanks!

Reply:
Did mypy stopped complaining about those classes just moving the to global context?

Reply:
Yep. And it was an internal error, not a normal error... didn't look into it though.